CPPLogger/CPPLogger.h

```diff
@@ -22,31 +22,37 @@ class ILogger
 public:
 	virtual ILogger& operator() (const std::string& func_name, const std::string& file_name, short level) = 0;
 	virtual ILogger& operator<< (const char* log_message) = 0;
-	virtual void set_filter_level(LogLevels level) = 0;
+	virtual void set_filter_level(unsigned int level) = 0;
 };
 
 class Logger final : public ILogger
 {
 public:
-	Logger();
 	~Logger();
 
+	static Logger* GetInstance();
+
 	Logger& operator()(const std::string& func, const std::string& filename, short level) override;
 	Logger& operator<<(const char* log_message) override;
 
-	void set_filter_level(LogLevels level) override;
+	void set_filter_level(unsigned int level) override;
 
 private:
+	Logger();
+	Logger(const Logger&);
+	
+	static Logger*  instance;
```

Comment:
For static members name should be:
s_instance;

Reply:
Fixed